src/amr/messengers/hybrid_hybrid_messenger_strategy.h

```diff
@@ -181,7 +181,6 @@ namespace amr
             magneticInit_.regrid(hierarchy, levelNumber, oldLevel, initDataTime);
             electricInit_.regrid(hierarchy, levelNumber, oldLevel, initDataTime);
             interiorParticles_.regrid(hierarchy, levelNumber, oldLevel, initDataTime);
-            levelGhostParticlesOld_.regrid(hierarchy, levelNumber, oldLevel, initDataTime);
```

Comment:
do we want this still?

Reply:
wtf... I thought it was back in... :/

Reply:
rebasing deletes that line silently actually....